### ISSUE TITLE
PropertySortOrder issue with duplicate properties

### ISSUE THREAD

**Issue by ndonohoe-bjss** [brigade/scss-lint#675]
There is a strange issue with PropertySortOrder when a block contains duplicated properties, the following block

```scss
.timepicker-times {
  position: absolute;
  top: -150px;
  top: -15vh;
  transform: scaleY(0);
  transition: transform 0.3s $ease-out-quint;
  z-index: 99;
  outline: none;
  background-color: $color-grey-50;
  width: 100%;
  max-height: 300px;
  max-height: 30vh;
  overflow-y: scroll;
}
```

Results in the following error, even though the order is correct
```
PropertySortOrder: Properties should be ordered position, top, top, transform, transition, z-index, outline, background-color, width, max-height, max-height, overflow-y
```

Config is:
```
  PropertySortOrder:
    enabled: true
    order: concentric
```

Removing both declarations of top removes the error, as does removing one declaration of max-height or the declaration of overflow-y.

scss-lint version is 0.43.2

**Comment by michaelhogg** [comment]
I'm experiencing this issue in [0.43.2](https://github.com/brigade/scss-lint/releases/tag/v0.43.2) too. It also occurs in [0.40.1](https://github.com/brigade/scss-lint/releases/tag/v0.40.1).

Linting this SCSS:

```scss
.testing {
    position:         absolute;
    top:              0;
    transition:       -webkit-transform .33s;
    transition:       transform .33s;
    background-color: hsl(0, 0%, 100%);
}
```
with this config:
```yaml
PropertySortOrder:
    enabled: true
    order: concentric
```
results in this error:
```
PropertySortOrder: Properties should be ordered position, top, transition, transition, background-color
```
Removing one or more of these properties from the SCSS prevents the error: `position`, `top`, `background-color`.

**Comment by sds** [comment]
Thanks for the reports, @ndonohoe-bjss / @michaelhogg.

Fixed in 4ea7ed1baca52abc64085b39a063d40e6d604571.
